test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs

```diff
@@ -219,7 +218,7 @@ internal async Task CheckUserExistenceInGroupTest()
             var token = serviceManager.GenerateClientAccessToken(hubName, user);
             using (StartVerifiableLog(out var loggerFactory, LogLevel.Debug))
             {
-                var serviceHubContext = await serviceManager.CreateHubContextAsync(hubName, loggerFactory);
+                var serviceHubContext = await serviceManager.CreateHubContextAsync(hubName, default);
```

Comment:
shall we keep the previous test code as it is and make sure there is no regression after code changes? (Or the changes are acceptable)

Reply:
fixed.